.github/workflows/build_test_push.yaml

```diff
@@ -80,6 +83,22 @@ jobs:
             MININORCAL_OSM_PATH=${{ secrets.MININORCAL_OSM_PATH}}
             MININORCAL_GTFS_PATH=${{ secrets.MININORCAL_GTFS_PATH}}
 
+      # Build the dev server image used in functional tests
```

Comment:
i think a comment here about why we're doing this step in a separate workflow from the functional test would be useful

Reply:
good call, added!